src/pages/Popup/components/Settings.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-duplicate-props */
```

Comment:
Do not disable lints
You can use prettier as we are also using it

Reply:
Updated.